### ISSUE TITLE
a11y: fix aria-required-parent

### ISSUE THREAD

**Issue by codyrancher** [rancher/dashboard#19154]
Accessibility violations this issue fixes, from the axe baseline ([`baseline.json`](https://github.com/rancher/dashboard/blob/5ffbdf11e010fde32ce1009eb6371b9c5c27e425/cypress/e2e/tests/accessibility/baseline.json)).

| Severity | Rule | Description | Count |
|---|---|---|---|
| 🔴 Critical | [`aria-required-parent`](https://dequeuniversity.com/rules/axe/4.10/aria-required-parent) | Ensure elements with an ARIA role that require parent roles are contained by them | 4 |
| | | **Total** | **6** |

Part of #17279.

**Comment by codyrancher** [comment]
Closing, this is largely covered by https://github.com/rancher/dashboard/issues/19119.

I'll reopen or open something new if that issues doesn't resolve these.

**Comment by codyrancher** [comment]
I decided to just reduce the issue to one violation type so there wasn't overlap.
